react-native/package.json

```diff
@@ -8,22 +8,26 @@
     "start": "react-native start",
     "test": "jest",
     "lint": "eslint .",
-    "prettier": "prettier --write src/*"
+    "prettier": "prettier --write src/*",
+    "generate:prod-partition-based-sync": "npx bluehawk copy . --output=../generated/react-native-partition-based-sync --ignore=realm.json --state=partition-based-sync",
+    "generate:prod-flexible-sync": "npx bluehawk copy . --output=../generated/react-native-flexible-sync --ignore=realm.json --state=flexible-sync",
+    "generate:prod": "npm run generate:prod-partition-based-sync && npm run generate:prod-flexible-sync"
```

Comment:
Good idea to have these bluehawk commands in the package.json, but I suggest we shorten the names:
- generate-both # since it generates both FS and PBS; or alternatively 'generate-all' or just 'generate'
- generate-pbs
- generate-fs

Reply:
I'm using the same convention here as we have in the [`web-js`](https://github.com/mongodb-university/realm-template-apps/blob/main/web-js/package.json#L28) template but I agree we could actually be a little simpler here. I don't want to use the shorter names because these map to the Bluehawk state names to keep everything simpler.

Gonna go with

- `npm run generate` - generate all of this project's artifact repos
- `npm run generate:<state>` - generate a specific artifact repo